src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -174,8 +174,6 @@ export default {
       this.shape.listenToOnce(this.paper, 'cell:pointerup blank:pointerup', () => {
         this.moveBoundaryEventIfOverTask();
         this.resetInvalidTarget();
-        this.$emit('save-state');
```

Comment:
Does this cause the undo/redo to ignore when a boundary event is created and attached?